nrf_rpc/include/nrf_rpc_common.h

```diff
@@ -141,6 +141,32 @@ extern "C" {
 			*(const uint8_t *const *)(_array_ptr);		       \
 		(_var) = (_type *)(_var) + 1, (void)_it
 
+/** @brief Get item from the array.
+ *
+ * @param _array_ptr Pointer to array.
+ * @param _index     Index of the item.
+ * @param _type      Type of items in array.
+ */
+#define NRF_RPC_AUTO_ARR_GET(_array_ptr, _index, _type)			       \
+	(((_type *)((const uint8_t *const *)(_array_ptr) + 1))[_index])
+
+/* Error codes definitions if they are not defined by the platform */
+#ifndef EIO
+#define EIO 5
+#endif
+#ifndef EINVAL
+#define EINVAL 22
+#endif
+#ifndef ENOMEM
+#define ENOMEM 12
+#endif
+#ifndef EPROTO
+#define EPROTO 71
```

Comment:
This may be a problem during sending codes between cores if we have different OS.
On a single side we will have always the same error numbers and it's not a problem.

I can add translating function before sending an error code. It can be empty if OS error codes are matching codes that we assumed inside the packets.

Reply:
I switched from `EPROTO` to `EBADMSG` which has no such compatibility problems.